tests/queries/0_stateless/01810_max_part_removal_threads_long.sh

```diff
@@ -30,7 +30,7 @@ $CLICKHOUSE_CLIENT -nm -q """
 
     -- sometimes the same thread can be used to remove part, due to ThreadPool,
     -- hence we cannot compare strictly.
-    select throwIf(not(length(thread_ids) between 6 and 11))
+    select throwIf(not(length(thread_ids) between 1 and 11))
```

Comment:
Using `1` here, means that it does not uses multiple threads, so the test is simply useless in this case.
Better to rewrite it to retry loop then.